velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -155,5 +155,61 @@ TEST_F(ProbabilityTest, binomialCDF) {
       binomialCDF(-2, 0.5, -1), "numberOfTrials must be greater than 0");
 }
 
+TEST_F(ProbabilityTest, inverseChiSquaredCDF) {
+  const auto invChiSquaredCDF = [&](std::optional<double> df,
+                                    std::optional<double> p) {
+    return evaluateOnce<double>("inverse_chi_squared_cdf(c0, c1)", df, p);
+  };
+
+  EXPECT_EQ(0.0, invChiSquaredCDF(3, 0.0));
+  EXPECT_EQ(1.4236522430352796, invChiSquaredCDF(3, 0.3));
+  EXPECT_EQ(11.344866730144370, invChiSquaredCDF(3, 0.99));
+  EXPECT_EQ(12.287969413016436, invChiSquaredCDF(7.2, 0.90));
+  EXPECT_EQ(37.714228443623732, invChiSquaredCDF(25.05, 0.95));
```

Comment:
We can remove these last 2 test cases (unless they are serving some special reason).

Reply:
Removed.